fbw-a32nx/src/systems/instruments/src/PFD/AttitudeIndicatorHorizon.tsx

```diff
@@ -55,7 +55,7 @@ class HeadingBug extends DisplayComponent<{ bus: ArincEventBus, isCaptainSide: b
         });
 
         sub.on('heading').handle((h) => {
-            this.heading = h;
+            this.heading = h.value;
```

Comment:
Seems a backwards step to discard the SSM, and in this case it should check the SSM.